<ui version="4.0" >
 <class>Form3</class>
 <widget class="QWidget" name="Form3" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>425</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>425</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Ustawienia</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <widget class="QCheckBox" name="enabledB" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Włącz wtyczkę</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyB" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>310</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Zastosuj</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="extsB" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>30</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Obsługiwane rozszerzenia plików</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>60</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>Samplerate</string>
   </property>
   <widget class="QRadioButton" name="hz44" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>44100Hz</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hz22" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>22050Hz</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hz11" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>11025Hz</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hz8" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>8000Hz</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>60</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>Resampling</string>
   </property>
   <widget class="QRadioButton" name="r1B" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Nearest</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="r2B" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Linear</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="r3B" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Spline</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="r4B" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Fir</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>180</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title" >
    <string>Surround</string>
   </property>
   <widget class="QSpinBox" name="surround2B" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>70</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Surround delays</string>
    </property>
    <property name="minimum" >
     <number>5</number>
    </property>
    <property name="maximum" >
     <number>40</number>
    </property>
    <property name="value" >
     <number>30</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="surround1B" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Surround level</string>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="value" >
     <number>50</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="surroundB" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>14</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>On</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>180</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title" >
    <string>Reverb</string>
   </property>
   <widget class="QSpinBox" name="reverb1B" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Reverb level</string>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="value" >
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="reverb2B" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>70</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Reverb delay</string>
    </property>
    <property name="minimum" >
     <number>40</number>
    </property>
    <property name="maximum" >
     <number>200</number>
    </property>
    <property name="value" >
     <number>100</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="reverbB" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>14</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>On</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>180</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title" >
    <string>Bass</string>
   </property>
   <widget class="QSpinBox" name="bass1B" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>XBass level</string>
    </property>
    <property name="minimum" >
     <number>0</number>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="value" >
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="bass2B" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>70</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>XBass cutoff in Hz</string>
    </property>
    <property name="minimum" >
     <number>10</number>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="value" >
     <number>50</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="bassB" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>14</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>On</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>30</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title" >
    <string>Inne opcje dźwięku</string>
   </property>
   <widget class="QCheckBox" name="loopB" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Loop</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bitsB" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>16bit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="stereoB" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Stereo</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="osB" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Oversampling</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="nrB" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Noise reduction</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>290</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title" >
    <string>Ustaw suwak na:</string>
   </property>
   <widget class="QRadioButton" name="timeB" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Czas [s]</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="patternB" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Pattern</string>
    </property>
   </widget>
  </widget>
  <widget class="QSlider" name="volS" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>40</y>
     <width>16</width>
     <height>321</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Głośność</string>
   </property>
   <property name="minimum" >
    <number>1</number>
   </property>
   <property name="maximum" >
    <number>255</number>
   </property>
   <property name="value" >
    <number>255</number>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>enabledB</tabstop>
  <tabstop>extsB</tabstop>
  <tabstop>volS</tabstop>
  <tabstop>hz44</tabstop>
  <tabstop>hz22</tabstop>
  <tabstop>hz11</tabstop>
  <tabstop>hz8</tabstop>
  <tabstop>r1B</tabstop>
  <tabstop>r2B</tabstop>
  <tabstop>r3B</tabstop>
  <tabstop>r4B</tabstop>
  <tabstop>osB</tabstop>
  <tabstop>nrB</tabstop>
  <tabstop>stereoB</tabstop>
  <tabstop>bitsB</tabstop>
  <tabstop>loopB</tabstop>
  <tabstop>bassB</tabstop>
  <tabstop>bass1B</tabstop>
  <tabstop>bass2B</tabstop>
  <tabstop>reverbB</tabstop>
  <tabstop>reverb1B</tabstop>
  <tabstop>reverb2B</tabstop>
  <tabstop>surroundB</tabstop>
  <tabstop>surround1B</tabstop>
  <tabstop>surround2B</tabstop>
  <tabstop>timeB</tabstop>
  <tabstop>patternB</tabstop>
  <tabstop>ApplyB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
